scripts/memcache-stats.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 argument="$1"
 
-IP_ADDR=$(hostname -I | sed 's/ /\n/g' | grep '10\.')
+IP_ADDR=$(hostname -I | grep -oP "\b10(\.\d+){3}" )
```

Comment:
@danielbeardsley that what you were going for?

I don't think we'll take any more "strokes" off playing regex golf.

---

This isn't exactly sane anyhow, but it'll work for our purposes.

I was planning on using the `/etc/hosts` file for now to address the memcached instances. **But**, if later we decide to use ENI's, machines could have _two_ `10.X` addresses and this would give... multiline output?

It's probably not worth over-analyzing until it's a problem...